### ISSUE TITLE
`mod_videoservice` link in `usage.md` does not work

### ISSUE THREAD

**Issue by TheHllm** [innocampus/videbo#1]
The link in to [mod_videoservice](https://github.com/innocampus/moodle-mod_videoservice) mentioned in [usage.md](https://github.com/innocampus/videbo/blob/master/docs/usage.md) does not work.

**Comment by daniil-berg** [comment]
You are right @TheHllm. I apologize for the confusion. I included the link assuming `mod_videoservice` will be publicly available soon. The project is still maintained privately. A few things came up and caused a delay. We will publish the plugin under that link as soon as possible. I expect within the first couple of weeks of next year.
